src/main/java/org/opensearch/ad/settings/AnomalyDetectorSettings.java

```diff
@@ -789,4 +789,18 @@ private AnomalyDetectorSettings() {}
     // Cold start setting
     // ======================================
     public static int MAX_COLD_START_ROUNDS = 2;
+
+    // ======================================
+    // Validate Detector API setting
+    // ======================================
+    public static final long TOP_VALIDATE_TIMEOUT_IN_MILLIS = 60_000;
+    public static final long MAX_INTERVAL_REC_LENGTH_IN_MINUTES = 120L;
+    public static final double INTERVAL_RECOMMENDATION_INCREASING_MULTIPLIER = 1.2;
+    public static final double INTERVAL_RECOMMENDATION_DECREASING_MULTIPLIER = 0.8;
+    public static final double INTERVAL_BUCKET_MINIMUM_SUCCESS_RATE = 0.75;
+    public static final double CONFIG_BUCKET_MINIMUM_SUCCESS_RATE = 0.25;
```

Comment:
Why do we have different sparsity requirement for interval and other configs?

Reply:
The interval recommendation is done with multiple different intervals while the A/B config testing is done only with the given interval. We previously discussed to lower this to 0.5 but I tried 0.25 and it seemed to make more sense as A/B testing was more about finding if config would significantly lower success rate. This is to make knowing which config cause the issue slightly more likely to be accurate but I can change this back to 0.5. Future implementation for 1.4 release could include changing implementation to check for multiple intervals for each configuration.

Reply:
I don't quite follow. So you start with interval search and find none can give you >= 75%. Then you go to search other config and try to find any one config results in <= 25%. That config/raw data is the potential issue.  If all of the following checks return a value < 75% > 25%, you would give no validation error/recommendation. Is that correct? If yes, in this case I would prefer you tell me my raw data is too sparse instead of no error.

Reply:
Yes there is the case where everything else is above 25% but below the 75% with everything.  I'll change it to suggest raw data is too sparse